src/main/java/com/example/Customer.java

```diff
@@ -0,0 +1,14 @@
+package com.example;
+
+/**
+ * A customer is a person who visits a shop for buying goods.
+ * <p>
+ * The customer can have a loyalty card that has a specific number.
+ */
+public final class Customer {
+    private final LoyaltyCard card;
```

Comment:
Please put Javadoc here to. Explain why does a customer have a card? 

What happens if a customer does not have a card? Suppose, I visit the shot for the very first time. I surely do not have a card, but I want to purchase a bottle of water because I'm thirsty. Can I?

How does the code behave in such a case?

Also, what happens if I do have a card, but forgot it at my home?

When do we _really_ need the card?